docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+  version: '3'
+
+  services:
+    db:
+      image: mysql
```

Comment:
Please always use a fixed image version, otherwise, some bugs may happen when a new version of the image is released.